src/dev/run_with_node_options/run_with_node_options.js

```diff
@@ -0,0 +1,93 @@
+/*
```

Comment:
Curious, why not just use a bash script like we do with `bin/kibana`? One thought I had recently is that we should move the current `bin` directory to the build resources as folks often mistakenly use them when running from source.

Reply:
I think that could work! If we move the scripts out of the bin to the build folder and just create a `bin/node` and a `bin/node.bat` we would be able to achieve the same end result. I think we can even replace use from `node scripts` to be `bin/node scripts` in the `package.json`. The only exception for this would be node_modules binaries like `gulp` or `grunt`. The only way to get around this if we want / felt that need would be create our own programatic node scripts to run things with those and then just call `bin/node scripts/gulp ...`. What do you think @tylersmalley ? I can try to achieve this in that PR, I think it makes sense.